doc/classes/Environment.xml

```diff
@@ -86,7 +86,7 @@
 			This is useful to simulate [url=https://en.wikipedia.org/wiki/Aerial_perspective]aerial perspective[/url] in large scenes with low density fog. However, it is not very useful for high-density fog, as the sky will shine through. When set to [code]1.0[/code], the fog color comes completely from the [Sky]. If set to [code]0.0[/code], aerial perspective is disabled.
 		</member>
 		<member name="fog_density" type="float" setter="set_fog_density" getter="get_fog_density" default="0.01">
-			The exponential fog density to use. Higher values result in a more dense fog.
+			The [i]exponential[/i] fog density to use. Higher values result in a more dense fog. Fog rendering is exponential as in real life. This means fog will never [i]fully[/i] obscure objects regardless of distance.
```

Comment:
Is this a helpful addition? From a mathematical point of view its true, but practically speaking if you turn density up past 1 and move more than a few meters away, you won't see anything at all.